tests/src/JIT/HardwareIntrinsics/X86/Avx/Add.Double.cs

```diff
@@ -77,26 +86,30 @@ private static void AddDouble()
     public sealed unsafe class SimpleBinaryOpTest__AddDouble
     {
         private const int VectorSize = 32;
-        private const int ElementCount = VectorSize / sizeof(Double);
 
-        private static Double[] _data1 = new Double[ElementCount];
-        private static Double[] _data2 = new Double[ElementCount];
+        private const int Op1ElementCount = VectorSize / sizeof(Double);
+        private const int Op2ElementCount = VectorSize / sizeof(Double);
+        private const int RetElementCount = VectorSize / sizeof(Double);
+
+        private static Double[] _data1 = new Double[Op1ElementCount];
+        private static Double[] _data2 = new Double[Op2ElementCount];
 
         private static Vector256<Double> _clsVar1;
         private static Vector256<Double> _clsVar2;
 
         private Vector256<Double> _fld1;
         private Vector256<Double> _fld2;
 
-        private SimpleBinaryOpTest__DataTable<Double> _dataTable;
+        private SimpleBinaryOpTest__DataTable<Double, Double, Double> _dataTable;
 
         static SimpleBinaryOpTest__AddDouble()
         {
             var random = new Random();
 
-            for (var i = 0; i < ElementCount; i++) { _data1[i] = (double)(random.NextDouble()); _data2[i] = (double)(random.NextDouble()); }
-            Unsafe.CopyBlockUnaligned(ref Unsafe.As<Vector256<Double>, byte>(ref _clsVar1), ref Unsafe.As<Double, byte>(ref _data2[0]), VectorSize);
-            Unsafe.CopyBlockUnaligned(ref Unsafe.As<Vector256<Double>, byte>(ref _clsVar2), ref Unsafe.As<Double, byte>(ref _data1[0]), VectorSize);
+            for (var i = 0; i < Op1ElementCount; i++) { _data1[i] = (double)(random.NextDouble()); }
```

Comment:
Need to handle the get next value separately for each operand, so that they can be appropriately filled when different.